MapboxMobileEvents/Categories/NSString+MMEVersions.m

```diff
@@ -106,44 +106,45 @@ - (NSInteger)mme_semverPatchVersion {
     return patchVersion;
 }
 
-// MARK: - Build Identifiers
-
-- (BOOL) mme_containsBuildStringComponent {
-    NSTextCheckingResult *regexMatch = [NSString.mme_semverExpression firstMatchInString:self options:0 range:NSMakeRange(0,self.length)];
-    return (regexMatch && regexMatch.numberOfRanges >= MMESemverBuildIndex);
-}
-
-- (NSString *)mme_semverBuildStringComponent {
-    NSString *buildString = nil;
-    NSTextCheckingResult *regexMatch = [NSString.mme_semverExpression firstMatchInString:self options:0 range:NSMakeRange(0,self.length)];
-    if (regexMatch && regexMatch.numberOfRanges >= MMESemverBuildIndex) {
-        buildString = [self substringWithRange:[regexMatch rangeAtIndex:MMESemverBuildIndex]];
-    }
-    return buildString;
-}
-
-- (NSArray<NSString *>*)mme_semverBuildIdentifiers {
-    return [[self mme_semverBuildStringComponent] componentsSeparatedByString:@"."];
-}
-
-// MARK: - Pre-Release Identifiers
-
-- (BOOL) mme_containsPreReleaseComponent {
-    NSTextCheckingResult *regexMatch = [NSString.mme_semverExpression firstMatchInString:self options:0 range:NSMakeRange(0,self.length)];
-    return (regexMatch && regexMatch.numberOfRanges >= MMESemverReleaseIndex);
-}
-
-- (NSString *)mme_semverPreReleaseComponent {
-    NSString *buildString = nil;
-    NSTextCheckingResult *regexMatch = [NSString.mme_semverExpression firstMatchInString:self options:0 range:NSMakeRange(0,self.length)];
-    if (regexMatch && regexMatch.numberOfRanges >= MMESemverReleaseIndex) {
-        buildString = [self substringWithRange:[regexMatch rangeAtIndex:MMESemverReleaseIndex]];
-    }
-    return buildString;
-}
-
-- (NSArray<NSString *>*)mme_semverPreReleaseIdentifiers {
-    return [[self mme_semverPreReleaseComponent] componentsSeparatedByString:@"."];
-}
+//TODO: These methods are not working as expected in tests and are not used currently
```

Comment:
IIRC there aren't tests for these, we should add some if we're going to keep the code at all (given it's not currently used, it could be deleted from the project until it's needed)

Reply:
When writing a couple of the tests it wasn't behaving as I would have expected. Removing the unused code for now seems like a good bet.